jmxtrans-core/src/main/java/com/googlecode/jmxtrans/jobs/ServerJob.java

```diff
@@ -3,6 +3,7 @@
 import com.googlecode.jmxtrans.connections.JMXConnectionParams;
 import com.googlecode.jmxtrans.jmx.JmxUtils;
 import com.googlecode.jmxtrans.model.Server;
+import com.sun.tools.attach.VirtualMachine;
```

Comment:
:+1:  good catch, we have to isolate this in a tiny class that would only be instantiated if Class.forname("com.sun.tools.attach.VirtualMachine") succeeds. Otherwise, jmxtrans may no longer start on IBM JVM ...